File: AdisyonUygulamaKotlinXMLsiz/app/src/main/java/com/example/adisyonuygulamakotlin/model/Urun.kt
package com.example.adisyonuygulamakotlin.model

data class Urun(
    val id: Int,
    var urun_ad: String,
    var urun_fiyat: Float,
    var urun_resim: String,
    var urun_adet: Int,
    var urunKategori: Kategori
)

File: adisyon_uygulama_react_native/src/model/urun.ts
import {Kategori} from "./kategori";

export type Urun = {
    id: number;
    urun_ad: string;
    urun_fiyat: number;
    urun_resim: string;
    urun_adet: number;
    urunKategori: Kategori;
};